packages/modules/viewer-react/src/components/iModel/IModelViewer.tsx

```diff
@@ -32,6 +32,10 @@ export const IModelViewer: React.FC<ModelProps> = ({
   frontstages,
   backstageItems,
 }: ModelProps) => {
+  const backstageItemsProvided = UiItemsManager.getBackstageItems();
+  const hasBackstageItems = backstageItemsProvided || (backstageItems &&
```

Comment:
look at the the type for backstageItemsProvided;
also, hasBackstageItems can be simplified